sale_loyalty_order_suggestion_multi_product/wizard/sale_loyalty_reward_wizard.py

```diff
@@ -60,7 +60,12 @@ def _compute_loyalty_rule_line_description(self):
         )
         if self.selected_reward_id and not self.applicable_program and products:
             if len(products) > 1:
-                products_str = f"{', '.join(map(str, [product.name for product in products][:-1]))} {_('and')} {[product.name for product in products][-1]}"  # noqa: B950
+                product_names = [product.name for product in products]
```

Comment:
```suggestion
                product_names = products.mapped("name")
```

And previously, `self.selected_reward_id.program_id.rule_ids.loyalty_criteria_ids.product_ids` already returns not duplicated products, so no need of `set` and then `list`.

Reply:
done, thanks